api/api-reference.md

```diff
@@ -0,0 +1,849 @@
+---
+title: Tiger Cloud REST API reference
+excerpt: A comprehensive RESTful API for managing Tiger Cloud resources including VPCs, services, and read replicas.
+tags: [REST]
+products: [cloud]
+---
+
+# $REST_LONG reference
+
+A comprehensive RESTful API for managing $CLOUD_LONG resources including VPCs, services, and read replicas.
+
+## Overview
+
+**API Version:** 1.0.0  
+**Base URL:** `https://console.cloud.timescale.com/public/api/v1`
+
+## Authentication
+
+The $REST_LONG uses HTTP Basic Authentication. Include your access key and secret key in the Authorization header.
+
+### Basic Authentication
+```http
+Authorization: Basic <base64(access_key:secret_key)>
+```
+
+### Example
+```bash
+# Using cURL
+curl -X GET "https://console.cloud.timescale.com/public/api/v1/projects/{project_id}/services" \
+  -H "Authorization: Basic $(echo -n 'your_access_key:your_secret_key' | base64)"
+```
+
+## Service Management
+
+You use this endpoint to create and manage the following Tiger Postgres services:
+
+- `TIMESCALEDB`: a Tiger Postgres instance optimized for real-time analytics service For time-stamped data like events,
```

Comment:
Something not working in this sentence. Missing a full stop somewhere?

Reply:
This is generated from source by engineering. We will update on the next round.